src/EventService.Data.Provider.MsSql.Ef/Migrations/20230126193800_InitialCreate.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using EventService.Data.Provider.MsSql.Ef;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.EventService.Data.Provider.MsSql.Ef.Migrations
+{
+  [DbContext(typeof(EventServiceDbContext))]
+  [Migration("20230126193800_InitialCreate")]
+  public class _20230126193800_InitialCreate : Migration
```

Comment:
rename it to InitialCreate, like here https://github.com/LT-Students/DigitalOffice-PositionService/blob/develop/src/PositionService.Data.Provider.MsSql.Ef/Migrations/20220522164300_InitialMigration.cs